code/modules/reagents/reagent_dispenser.dm

```diff
@@ -15,6 +15,10 @@
 	var/can_be_tanked = TRUE
 	///Is this source self-replenishing?
 	var/refilling = FALSE
+	///Can this dispenser be opened using a wrench?
+	var/openable = TRUE
```

Comment:
I know this is a nitpick but make this by default false, this should be an opt-in mechanic for subtypes